docs/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+# Description
+
+Please include a summary of the changes and the related issue, if it exists. 
+Also, include relevant motivation and context for this PR. List any
+dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How has this been tested?
+
+Please describe the tests that you wrote to verify your changes. //FIXME: Do we want this?
```

Comment:
Since in doubt, we could remove this for the moment, and add something about `How has this been tested?` when we feel the need.

Usually tests should already be clear enough to be understood without much extra description, if some extra information is needed I'd see acceptable having that extra context in the description.

wdyt?

Reply:
Most of these can be implemented as checks in the CI, but some of them I find them at least valuable to maybe be included in the CONTRIBUTING.md

Reply:
Good, what is your proposal instead of the `FIXME` :smile: ?